integration/users/usersinfo_test.go

```diff
@@ -91,12 +108,14 @@ func TestUsersinfo(t *testing.T) {
 	}
 
 	testCases := map[string]struct { //nolint:vet // for readability
-		dbSuffix   string
-		payload    bson.D
-		err        *mongo.CommandError
-		altMessage string
-		expected   bson.D
-		hasUser    map[string]struct{}
+		dbSuffix        string
+		showCredentials bool // showCredentials should also be set on the payload
```

Comment:
They why we have a separate field instead of just using `payload.Get("showCredentials")`?

Reply:
I did so trying to simplify the tests, but I've refactored the test to get this from the payload directly.
The downside is that there is some extra complexity in retrieving the value of showCredentials, but it seems like a good trade-off.